recipes/nutch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nutch" %}
+{% set version = "1.10.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "17cf195db88b11c5d283448fe362d483e78db58dc564339583de629790103782" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
```

Comment:
Is this required? The docs on this are not very insightful:

https://conda.io/docs/building/meta-yaml.html#preserve-python-egg-directory